src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -364,6 +364,25 @@ users.
 Learn more by reading our complete [Delivering Fast and Light
 Applications with Save-Data] article.
 
+#### Network information
```

Comment:
Is there a better title that is more usecase led? Intelligent loading based on network information?

Reply:
I like it! Thanks!

Reply:
Addressed at https://github.com/google/WebFundamentals/pull/5058/commits/9854ec43241cb0773111b1cc05bc6b0c28d0ca19